Makefile

```diff
@@ -529,7 +543,15 @@ prepare-release: ## Generates a makefile that will override environment variable
 	VERSION=$(VERSION)\nREPLACES_VERSION=$(REPLACES_VERSION)" > $(RELEASE_FILE)
 	$(MAKE) bundle
 	$(MAKE) helm-build VERSION=$(VERSION)
+	$(MAKE) update-release-version VERSION=$(VERSION)
+
+.PHONY: read-version
+read-release-version: ## Reads release version
+	@$(SED) -n 's/^.*Version = "\([^"]*\)".*/\1/p' $(PROJECT_PATH)/internal/version/version.go
```

Comment:
I am considering having release version being injected into golang code with ld flags like `gitSHA` parameter. Thoughts @mikenairn ? This reading/parsing and updating golang code does not feel robust, it is prone to errors. Also, for reading version set, I did not want to run go code to avoid installing golang toolkit.

Reply:
I think this is a good idea

Reply:
I have no issue with this. It's just info, and doing it this way will be easier to deal with.